README.md

```diff
@@ -63,9 +63,9 @@ export default Ember.Route.extend({
 import ModelDecorator from 'ember-clothier/model-decorator';
 
 export default ModelDecorator.extend({
-  formatedDate: function() {
+  formatedDate: Ember.computed('createdOn', function() {
```

Comment:
Function prototype extensions are discourage at present, we have already rewritten the guides away from them.